renderer/containers/UI/FormattedDateTime.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { FormattedDate, FormattedTime } from 'react-intl'
+import { settingsSelectors } from 'reducers/settings'
+
+const mapStateToProps = state => ({
+  timeDisplayMode: settingsSelectors.currentConfig(state).timeDisplayMode,
+})
+
+const FormattedDateTime = ({ timeDisplayMode, format, month = 'long', value }) => {
+  const hour12 = timeDisplayMode && timeDisplayMode === '12hour' ? 1 : 0
```

Comment:
General style rule is that booleans should use format `isX` - I'd call this `is12Hour`

Reply:
If you set default value in props above this can be simplified to something like:

```js
const is12Hour = timeDisplayMode === '12hour'
```

Reply:
Done.